common/test/views/support/cleaner/AmpAdCleanerTest.scala

```diff
@@ -39,12 +39,19 @@ class AmpAdCleanerTest extends FlatSpec with Matchers {
 
   }
 
-  "AmpAdCleaner" should "only add 2 ads in total" in {
-    val doc = s"""<html><body>${s"<p>${tenChars * 70}</p>" * 4}</body></html>"""
+  "AmpAdCleaner" should "include ads at defined, regular intervals" in {
```

Comment:
might go for something like `"not have been changed without someone checking the tests are still valid"`